assets/js/pages/ClusterDetails/ClusterNodeName.jsx

```diff
@@ -6,31 +6,34 @@ import {
   EOS_POWER_OFF_OUTLINED,
 } from 'eos-icons-react';
 
+import { filter } from 'lodash';
+
 import Tooltip from '@common/Tooltip';
 import ClusterNodeLink from './ClusterNodeLink';
 
-const getNodeStatusIcon = (status) => {
-  switch (status) {
-    case 'Online': {
-      return <EOS_BOLT_FILLED className="tn-online" />;
-    }
-    case 'Offline': {
-      return <EOS_POWER_OFF_OUTLINED className="tn-offline" />;
-    }
-    case 'Maintenance': {
-      return <EOS_BUILD_OUTLINED className="tn-maintenance" />;
-    }
-    default: {
-      return <EOS_WARNING_OUTLINED className="tn-unknown" />;
-    }
-  }
+const statusIcons = {
+  Online: <EOS_BOLT_FILLED className="tn-online" />,
+  Offline: <EOS_POWER_OFF_OUTLINED className="tn-offline" />,
+  Maintenance: <EOS_BUILD_OUTLINED className="tn-maintenance" />,
+  Default: <EOS_WARNING_OUTLINED className="tn-unknown" />,
 };
 
-function ClusterNodeName({ status, hostId, children }) {
+const getNodeStatusIconAndMessage = (status, resources) => {
+  const unmanagedResources = filter(resources, { managed: false }).length;
+  const message =
+    unmanagedResources > 0
```

Comment:
This logic is not correct.
We need to follow the correct precedence to create the message:
```
offline -> warning icon (tooltip offline)
unclean -> warning icon (tooltip unclean)
standby -> warning icon (tooltip standby)
maintenance -> maintenance icon
resource unmanaged -> maintenance icon
```

This means that we need to have this tooltip if the previous states are not met.
That's why most probably the previous `switch/case` was a good idea, as it gives you this ordering.

Besides that, with this new logic the `x resources unmanaged` message wouldn't have the maintenance icon.